src/app/shared/forms/planet-markdown-textbox.scss

```
@@ -51,6 +51,93 @@ planet-markdown-textbox {
-  background: white !important;
+  background: v.$white !important;
+}
+
+// The component marks only the nearest owner/action pair and measures the row's rendered
+// height, so wrapped button rows still fit; when no action row is found it adds no classes
+// and fullscreen behaves as before.
+.planet-markdown-fullscreen-owner {
+  .planet-markdown-fullscreen-actions {
+    position: fixed;
+    inset: auto 0 0;
+    // Rows styled `width: 100%` would over-constrain the fixed box and drop `right`.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)